docs/content/_navigation.json

```diff
@@ -24,7 +24,7 @@
     ]
   },
   {
-    "title": "Tutorial",
+    "title": "Your first data pipeline",
```

Comment:
@sryza I tweaked this to be "Build your first pipeline" instead of "Your first data pipeline" - my thinking is that 'build' conveys it's a bit more involved than just a single code example. Opening the sidenav will display the individual sections that go into building the pipeline as well. Thoughts?

Reply:
I prefer to not use tutorial because of its ambiguity with guides. The 'build your first XX' is a lot more explicit as to the difference between that section and what is in the guides (which are going further than just getting started). Ultimately, I think this doesn't have to be a blocker. I support just getting this out there and if we find that people find it confusing, we can change the label with minimal addtl friction